content/engineering/tools/accessibility-scanning.md

```diff
@@ -57,17 +56,7 @@ Install pa11y locally:
 
 Create a `.pa11yci` in the root of your directory to [configure your pa11y CI run](https://github.com/pa11y/pa11y-ci#configuration). This step is optional, but helpful if you want to change any default configuration.
 
-For an example, take a look at the [18F accessibility site's .pallyci file](https://github.com/18F/accessibility/blob/18f-pages/.pa11yci/TODO/):
-
-```json
-    {
-      "defaults": {
-        "concurrency": 4,
-        "standard": "WCAG2AA",
-        "runners": ["axe"]
-      }
-    }
-```
+For an example, take a look at the [18F guides site's .pallyci file](https://github.com/18F/guides/blob/main/.pa11yci).
```

Comment:
The combined guides `.pa11yci` is different from the Accessibility Guide `.pa11yci` and will likely change over time. 

Pointing to the link and removing the specific config seems good, since the contents of the file will likely change.

Reply:
LGTM